masonry/src/lib.rs

```diff
@@ -141,9 +163,13 @@
 //! [masonry_winit]: https://crates.io/crates/masonry_winit
 //! [Xilem]: https://github.com/linebender/xilem/tree/main/xilem
 //! [tracing_tracy]: https://crates.io/crates/tracing-tracy
-
-// TODO: Add screenshot. This can't use include_screenshot as that doesn't work with cargo-rdme
-// See https://github.com/linebender/xilem/issues/851
+//!
+#![cfg_attr(
+    not(docsrs),
+    doc = "**Warning: This documentation is meant to be read on docs.rs. Screenshots may fail to load otherwise.**\n\n"
+)]
+#![doc = util::include_screenshot_reference!("to-do-screenshot", "example_to_do_list_initial.png")]
```

Comment:
We should have a comment here noting that this isn't a screenshot generated by the doc test.

Of course, ideally, we would just generate it in the doc test, but that's not for this PR.

Reply:
> Of course, ideally, we would just generate it in the doc test, but that's not for this PR.

I think the doc example is already too big for something that shows up in the main README.

Reply:
Fwiw, this would probably still be `#`ed out, i.e. not copied by cargo rdme or shown by rustdoc. The example as it currently stands is likely too big, but having it automatically tested shouldn't add to that.